library/src/main/java/com/datatorrent/lib/window/WindowOption.java

```diff
@@ -0,0 +1,143 @@
+package com.datatorrent.lib.window;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ *
+ */
+public abstract class WindowOption
```

Comment:
Does it make sense to define a merge() function (like in the DataFlow paper) which combines multiple overlapping windows into one?